README.md

```diff
@@ -8,13 +8,52 @@ Introducing the nostr-java library, a solution written in java for generating, s
 - Java 19+
 
 ## Usage
-To use the library in your project, add the following dependency to your pom.xml file:
+To use the library in your project, add the following dependency to your pom.xml file.
+
+Define the value for the most recent version available:
+```xml
+    <properties>
+        <nostr-java.version>0.6.1-SNAPSHOT</nostr-java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+```
+
+Add Jitpack to the repositories:
```

Comment:
Obtaining nostr-java library jars.  Two options:

Option 1: For pre-compiled binaries, add Jitpack to pom.xml repositories:
```xml
    <repositories>
        <repository>
            <id>jitpack.io</id>
            <url>https://jitpack.io</url>
        </repository>
    </repositories>
````
Option 2: To build and install your own nostr-java dependency libraries:
```
$ cd <your_git_home_dir>
$ git clone git@github.com:tcheeric/nostr-java.git
$ cd nostr-java
$ git checkout <relevant_branch>
$ mvn clean install
```